### PR TITLE
feat: add local wifi config

### PR DESCRIPTION
**Pull request by Walt-Intel**

### CODE REVIEW

**Comment by madhavilosetty-intel** [comment]
We just need one config file, and one secrets file that are generic to all functionalities.

File: internal/flags/activate.go [hunk 1]
```diff
@@ -21,7 +21,7 @@ func (f *Flags) handleActivateCommand() int {
 		return nil
 	})
 	// for local activation in ACM mode need a few more items
-	f.amtActivateCommand.StringVar(&f.configContent, "config", "", "specify a config file ")
+	f.amtActivateCommand.StringVar(&f.configContent, "configFile", "", "specify a config file ")
```
Comment (madhavilosetty-intel) [inline internal/flags/activate.go:24]: I would prefer to leave the flag as config, so in the future, it would accept either string or file with our ongoing tech debt task.

File: internal/flags/info.go [hunk 2]
```diff
@@ -6,16 +6,18 @@ import (
...
-	Lan      bool
-	Hostname bool
+	Ver        bool
+	Bld        bool
+	Sku        bool
+	UUID       bool
+	Mode       bool
+	DNS        bool
+	Cert       bool
+	UserCert   bool
```
Comment (madhavilosetty-intel) [inline internal/flags/info.go:16]: Please remove this user and system cert as it belongs to different feature changes

File: internal/local/info.go [hunk 3]
```diff
@@ -3,15 +3,18 @@ package local
...
 	"rpc/pkg/utils"
 	"strconv"
 	"strings"
+
+	log "github.com/sirupsen/logrus"
 )
 
 func (service *ProvisioningService) DisplayAMTInfo() int {
+	service.setupWsmanClient("admin", "Intel123!")
+	// service.setupWsmanClient("admin", service.flags.Password)
```
Comment (rsdmike) [inline internal/local/info.go:17]: Need to remove hard coded password

File: internal/local/info.go [hunk 4]
```diff
@@ -153,30 +156,52 @@ func (service *ProvisioningService) DisplayAMTInfo() int {
 		}
 	}
 	if service.flags.AmtInfo.Cert {
-		result, err := amtCommand.GetCertificateHashes()
-		if err != nil {
-			log.Error(err)
+		if service.flags.AmtInfo.SystemCert && !service.flags.AmtInfo.UserCert {
```
Comment (rsdmike) [inline internal/local/info.go:159]: let's move the cert stuff to another PR

**Review by rsdmike — CHANGES_REQUESTED** [review]

File: config.yaml [hunk 5]
```diff
@@ -13,3 +13,26 @@ acmactivate:
...
+    pskPassphrase: 'example123!@#'
+    ieee8021xProfileName: ''
+  - profileName: 'exampleWifi8021x' # friendly name (ex. Profile name)
+    ssid: 'ssid'
+    priority: 2
+    authenticationMethod: 7
+    encryptionMethod: 4
+    pskPassphrase: ''
+    ieee8021xProfileName: 'exampleIeee8021xEAP-TLS'
+ieee8021xConfigs:
```
Comment (madhavilosetty-intel) [inline config.yaml:31]: Remove ieee8021xConfigs at the beginning of the file to avoid redundancy.

File: secrets.yaml [hunk 6]
```diff
@@ -0,0 +1,7 @@
+secrets:
+  - profileName: 'wifiWPA2'
```
Comment (madhavilosetty-intel) [inline secrets.yaml:2]: These profile names should match the ones in the config file, but it's not clear whether they should refer to the Wi-Fi profile name or IEEE. It would be easier to figure out if we match the profile name in our example.

**Review by rsdmike — APPROVED** [review]
